stacks-common/src/libcommon.rs

```diff
@@ -59,4 +59,8 @@ pub mod consts {
     pub const MINER_REWARD_MATURITY: u64 = 100;
 
     pub const STACKS_EPOCH_MAX: u64 = i64::MAX as u64;
+
+    /// The number of StackerDB slots each signing key needs
+    ///  to use to participate in DKG and block validation signing.
+    pub const SIGNER_SLOTS_PER_USER: u32 = 11;
```

Comment:
Just want to confirm with @jferrant that this value is correct.

Reply:
It's just moved over from the existing constant -- it's easy enough to change in a follow up PR if necessary.

Reply:
It will become 12 soon. But its okay to be 11 right now. I can update it as needed.